test/Stateless.Tests/DotGraphFixture.cs

```diff
@@ -24,9 +24,8 @@ void OnExit()
         [Test]
         public void SimpleTransition()
         {
-            var expected = "digraph {" + System.Environment.NewLine
```

Comment:
Expected results are completely different. I ran each test and validated the output before commit